src/entities/Category/types.ts

```diff
@@ -5,4 +5,7 @@ export type CategoryAttributes = {
   updated_at: Date
 }
 
-export const ALL_PLACE_CATEGORY = "all"
+export type CategoryWithPlaceCount = {
+  name: string
+  count: string
```

Comment:
why a property named `count` is no a number?

Reply:
`pg` returns a string for `count(*)`. anyways I'll change it for `number` and cast it inside the model